code/iaas/model/src/main/java/io/cattle/platform/core/constants/InstanceConstants.java

```diff
@@ -4,7 +4,9 @@
 
     public enum SystemContainer {
         NetworkAgent,
-        LoadBalancerAgent;
+        LoadBalancerAgent,
+        ClusterDockerAgent, // socat
```

Comment:
As we only have one system instance for the cluster right now, only one enum value should be listed.